Widgets/widget.css

```diff
@@ -0,0 +1,61 @@
+
+body {
+   background-color:black;
+   font-family: 'Cantarell';
+   color:white;
+   border: solid #23c1e8 4px;
+   background-color: #194d6c;
+}
+
+/* Headers */ 
+
+h1 {
+   font-size: 2em;
+   font-weight:  bold;
+   margin: 0.2em;
+}
+
+h2 {
```

Comment:
Just as a note, this isn't used yet... I just wanted somewhere to save that green color